### PR TITLE
Added a few Array problems from Hackerrank and Codeforces

### PR DESCRIPTION
**Pull request by shreayan98c**

### CODE REVIEW

File: CodeChef_problems/Laddu Accural System/problem.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Laddu Accural System
+
+Problem: [https://www.codechef.com/laddu](https://www.codechef.com/laddu)
```
Comment (Vishruth-S) [inline CodeChef_problems/Laddu Accural System/problem.md:3]: The problem link seems to be incorrect.
Please correct it 😅

**Review by Vishruth-S — CHANGES_REQUESTED** [review]
Could you please add comments explaining the logic to  solutions of `Theatre Square` and `Largest Histogram`

File: CodeChef_problems/Laddu Accural System/problem.md (hunk 1 above)
Comment (shreayan98c) [reply CodeChef_problems/Laddu Accural System/problem.md]: I have fixed the link, as well as made the changes that were requested

**Comment by shreayan98c** [comment]
Added the explanations to those programs as well.

File: CodeChef_problems/Laddu Accural System/problem.md (hunk 1 above)
Comment (Vishruth-S) [reply CodeChef_problems/Laddu Accural System/problem.md]: The problem link you have given:https://www.codechef.com/laddu just leads to the Codechef laddu page and not any problem page. Could you please check again

Comment (shreayan98c) [reply CodeChef_problems/Laddu Accural System/problem.md]: Hi, I apologize for the issue. I realized the link leads to the homepage of the competition that had that question and not to the question directly itself. The question can be found at https://www.codechef.com/problems/LADDU , and I have fixed the link for the same in the repo. Again, I would like to apologize for the issue.

**Comment by shreayan98c** [comment]
I have updated the link, and it is working now.

File: CodeChef_problems/Laddu Accural System/problem.md (hunk 1 above)
Comment (Vishruth-S) [reply CodeChef_problems/Laddu Accural System/problem.md]: Oh it's alright 👍

**Review by Vishruth-S — APPROVED** [review]
Awesome 👍 Thanks for contributing 😄

**Pull request merged**